tests/test_density.py

```diff
@@ -0,0 +1,168 @@
+from argparse import ArgumentParser
+from parcels.field import Field
+from parcels.grid import Grid
+import numpy as np
+import math
+from parcels.particle import Particle, JITParticle
+
+
+def RK4(fieldx, fieldy, lon, lat, time, dt):
+    f_lat = dt / 1000. / 1.852 / 60.
+    f_lon = f_lat / math.cos(lat*math.pi/180)
+    u1 = fieldx[time, lon, lat]
+    v1 = fieldy[time, lon, lat]
+    lon1, lat1 = (lon + u1*.5*f_lon, lat + v1*.5*f_lat)
+    u2, v2 = (fieldx[time + .5 * dt, lon1, lat1], fieldy[time + .5 * dt, lon1, lat1])
+    lon2, lat2 = (lon + u2*.5*f_lon, lat + v2*.5*f_lat)
+    u3, v3 = (fieldx[time + .5 * dt, lon2, lat2], fieldy[time + .5 * dt, lon2, lat2])
+    lon3, lat3 = (lon + u3*f_lon, lat + v3*f_lat)
+    u4, v4 = (fieldx[time + dt, lon3, lat3], fieldy[time + dt, lon3, lat3])
+    Vx = (u1 + 2*u2 + 2*u3 + u4) / 6.
+    Vy = (v1 + 2*v2 + 2*v3 + v4) / 6.
+    return [Vx, Vy]
+
```

Comment:
So as I have been developing alternative kernels for the diffusion/random walk behaviours referenced in #68, I find myself having to interpolate through time using RK4 for a lot of different fields. It seems needless to continually re-write the routine substituting `U` and `V` for my field variables of interest, and so I've abstracted the RK4 slightly in this version- you just pass the x and y fields you wish to get the forcing from.

@mlange05, not sure if this is mucking up things on the JIT side, but it would be really useful to have a generic RK4 method that can be called for fields other than `U` and `V`.